tests/x11regressions/gnote/gnote_undo_redo.pm

```diff
@@ -37,15 +38,17 @@ sub run {
 
     #assure undo and redo take effect after save note and re-enter note
     send_key "ctrl-tab";    #jump to toolbar
-    send_key "ret";         #back to all notes interface
+    send_key "ctrl-tab" if (is_sle && sle_version_at_least('15'));    #jump to toolbar for SLED15
+    send_key "ret";                                                   #back to all notes interface
     send_key_until_needlematch 'gnote-new-note-matched', 'down', 6;
     wait_still_screen 3;
     send_key "ret";
     $self->undo_redo_once;
 
     #clean: remove the created new note
-    send_key "ctrl-tab";    #jump to toolbar
-    send_key "ret";         #back to all notes interface
+    send_key "ctrl-tab";                                              #jump to toolbar
+    send_key "ctrl-tab" if (is_sle && sle_version_at_least('15'));    #jump to toolbar for SLED15
```

Comment:
Is a ` send_key_until_needlematch 'ctrl-tab'` could not do the job here? Could be easier than using a version test (sorry, I didn't see this last time).

Reply:
Thank you for pointing this out. Actually I was considering use `send_key_until_needlematch 'ctrl-tab'`, and on the other hand I am trying to highlight the different behavior between SLED15 and earlier SLED version. It would be more clear to show the difference.
Please let me know if you still want to use `send_key_until_needlematch 'ctrl-tab'`  here.

Reply:
OK for me, will see if the merger will be ok with this. As I don't have merge rights, I'm only a young reviewer ;-)